src/pages/projectSettings/CreateDuplicateProjectButton.tsx

```diff
@@ -90,13 +90,15 @@ export const CreateDuplicateProjectButton: React.VFC<Props> = ({
           open={createModalOpen}
           owner={owner}
           repo={repo}
+          requestS3Creds={false}
         />
       )}
       <CopyProjectModal
         handleClose={() => setCopyModalOpen(false)}
         id={id}
         label={label}
         open={copyModalOpen}
+        requestS3Creds={false}
```

Comment:
It doesn't look like either of these components actually use this prop so no need to add it!

Reply:
removed i think (i dont know if i understand this completely sorry)